### DISCUSSION TITLE
Best way to have component(s) persists across all pages?

### DISCUSSION THREAD

**Discussion by gibumkang** [expo/router#644] [Q&A] [answered]
I have a component that I would like to have persistent across my pages once they are authenticated. It's a unique component that doesn't make sense to add to the header nor the tabs on the footer. Is there another way I can have this component on all the pages? Would I be OK adding this onto the associated `_layout` file?

**Comment by marklawlor** [comment, marked as answer]
Yes, this is the purpose of the `_layout` files
